PhotoService/ClientApp/src/components/layout-component/Layout.js

```diff
@@ -17,31 +17,29 @@ export default function Layout() {
   
 
   const [isLoggedIn, setIsLogged] = useState(authService.isLoggedIn())
-  const [userName, setUsername] = useState(authService.getOwnerUsername())
+  const [username, setUsername] = useState(authService.getOwnerUsername())
 
   const onLoggedChange = () => {
     setIsLogged(authService.isLoggedIn())
     setUsername(authService.getOwnerUsername())
   }
 
   return (
-    <>
-      <Navbar isLoggedIn={isLoggedIn} onLoggedChange={onLoggedChange} username={userName}/>
-      <div className='main-content'>
-        <Routes >
-          <Route exact path='/login' element={<LoginPage onLoggedChange={onLoggedChange} />} />
-          <Route exact path='/signup' element={<SignupPage onLoggedChange={onLoggedChange} />} />
-          <Route exact path='/verify' element={<VerificationPage />} />
-          <Route exact path='/image/:id' element={<ImageDetailsPage/>}/>
-          <Route exact path='/image/add' element={<ImageAddingPage/>}/>
-          <Route exact path="/profile/:userName" element={<Profile/>} >
-            <Route exact path=":tab" element={<Profile/>} />
-          </Route>
-          <Route exact path='/' element={<Home/>} />
-        </Routes>
-      </div>
-      <Footer/>
-    </>
+    <div>
+      <Navbar isLoggedIn={isLoggedIn} onLoggedChange={onLoggedChange} username={username} />
+      <Routes >
+        <Route exact path='/login' element={<LoginPage onLoggedChange={onLoggedChange} />} />
+        <Route exact path='/signup' element={<SignupPage onLoggedChange={onLoggedChange} />} />
+        <Route exact path='/verify' element={<VerificationPage />} />
+        <Route exact path='/image/:id' element={<ImageDetailsPage/>}/>
+        <Route exact path='/image/add' element={<ImageAddingPage/>}/>
+        <Route exact path="/profile/:username" element={<Profile/>} >
+          <Route exact path=":tab" element={<Profile/>} />
+        </Route>
+        <Route exact path='/' element={<Home/>} />
+      </Routes>
+      <Footer />
+    </div>
```

Comment:
wrap <Routes> witch <div className='main-content'> to provide correct footer layout